templates/vrrp_sync_group.erb

```diff
@@ -48,6 +48,14 @@ vrrp_sync_group <%= @_name %> {
   #     ("MASTER"|"BACKUP"|"FAULT")
   notify <%= @notify_script %>
   <%- end -%>
+  <%- if @track_interface -%>
```

Comment:
```suggestion
  <%- if @track_interface { -%>
```

I think there is a `{` missing?

Reply:
I don't think so, there's no need for it in an erb conditional.